nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractDatabaseFetchProcessor.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.nifi.processor.exception.ProcessException;
 import org.apache.nifi.processor.util.StandardValidators;
 import org.apache.nifi.processors.standard.db.DatabaseAdapter;
+import org.apache.nifi.processors.standard.db.impl.PhoenixDatabaseAdapter;
```

Comment:
We may end up moving the abstract class and interface(s) out to its own NAR so other folks can implement DB adapters outside the standard NAR. In that case we'd want to replace this with something like the contains() we did for Oracle. Also if there were a need for 2 separate Phoenix adapters, we'd have to do that anyway...

Reply:
Sounds good!